src/coreclr/interop/interoplib.cpp

```diff
@@ -206,4 +209,29 @@ namespace InteropLib
     }
 
 #endif // FEATURE_COMWRAPPERS
+#ifdef FEATURE_OBJCWRAPPERS
+    namespace ObjC
+    {
+        void GetOperatingSystemVersionFromNSProcessInfo(
```

Comment:
Does this need to be part of the runtime? Can this just be in the libraries shims instead?

Reply:
I could move this into the libraries shims if you prefer. I should be able to hook up System.Native to include this without issue.